Tools-Override/tests.targets

```diff
@@ -211,8 +211,54 @@
       <TestCommandLines Include="cd native"/>
     </ItemGroup>
 
+    <!-- For UAP F5 scenario, generate the test commands -->
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPRunnerFolder)'==''" Text="Please specify UAPRunnerFolder" />
+    <Error Condition="'$(BuildingUAPVertical)' == 'true' and '$(UAPLauncherFolder)'==''" Text="Please specify UAPLauncherFolder" />
```

Comment:
Can we figure out a way to make it such that we only have to set one folder path to enable tests? To make it easier. Similar to .NET Native you only need to set the TestILCPath.